### ISSUE TITLE
Feature request: customizable white space characters

### ISSUE THREAD

**Issue by LaurV2** [notepad-plus-plus/notepad-plus-plus#11835]
### Description of the Issue

As the title says, especially for space. Those for tab, LF, CR, etc, are not important, but the actual \big\cdot character for space is quite invasive, if you compare it, for example, with Keil's uVision (just a single pixed centered dot). In the older versions of Notepad++, the color was not customizable either, so the "view white space" was unusable, more confusing than helping. We salute the possibility to customize its color in the newer versions, as now we can make it very-very (I mean VERY) faint gray, almost white, so it becomes usable now. But we would honestly like a bit darker, smaller dot... 

### Steps to Reproduce the Issue

1. Turn on the "view white space"
2. 
3. 

### Expected Behavior

Proper, non-intrusive marking of the white space

### Actual Behavior

Your eyes hurt trying to distinguish the text from the big furballs on screen, unless (in the new versions) you edit the color profile to make the furballs very faint. 

### Debug Information

- not relevant

default coloring: 
(hurting doesn't it?) - if you forget some spaces there, you almost can't find your brackets or other characters
![default](https://user-images.githubusercontent.com/77255007/175472588-bdca533b-0c23-4623-96e5-5cf0450ca9cb.JPG)

after making the color style faint:
(in newer versions)
- usable, but still ugly, dots are too big, still confusing with real dots, or commas, etc
![faint](https://user-images.githubusercontent.com/77255007/175472597-ed6cf6a6-3bc0-40a4-ac43-39c03b104d9a.JPG)

sample of how it looks in uVision (sorry for the different color scheme)
- more customizable, easy to see what is white space and what not, not confusing with real dots, commas, etc, you can see lines running out (maybe some code at the end there, which will screw out your results :D) or the brackets lost somewhere...
![uVision](https://user-images.githubusercontent.com/77255007/175472600-f0298d5f-0cd6-4b8a-8f60-ac062f267a34.JPG)

**Comment by alankilborn** [comment]
If you are willing to use the PythonScript plugin, a small bit of code allows you to alter this yourself:

```
editor.setWhitespaceSize(1)
```
Change the `1` I show to suit your desires.

Your issue has been discussed in the past, likely some interesting reading (if you care) at these links:

* https://community.notepad-plus-plus.org/topic/13362/show-and-symbol-instead-of-cr-lf-when-whitespaces-enabled
* https://community.notepad-plus-plus.org/topic/13775/changing-colors-of-the-space-and-other-symbols-when-view-all-is-turned-on
* https://community.notepad-plus-plus.org/topic/14114/how-to-zoom-in-on-white-space-symbols
* https://community.notepad-plus-plus.org/topic/18082/increase-whitespace-size-macro-issue

**Comment by byzod** [comment]
I agree with @LaurV2 , it's possible to change with a script host but you need to install a correct plugin then add the script to startup script and change the lazy load mode. 

I mean, you CAN achieve what you want but that is much more complicated than changing an option in preference (if there's any), this is a minor problem but might helps a lot of people if this feature happens

I'm using 4k monitor and big font for a comfortable sitting posture, the whitespace dot is barely visible
![1](https://user-images.githubusercontent.com/848607/178135356-772feaee-fbec-4ac1-bb11-2bf5299ba856.png)

**Comment by wwwchriswww** [comment]
This indeed would be a nice feature improvement to have, i.e. have TAB and Space visuals controlled independently. For coding it would be nice to have the unobtrusive TAB arrow, without a dot standing in for space.
If you want to go the whole hog, so to speak, each non-printable character could be individually custom mapped, and have them grouped in some coherent groups, that are then turned on/off as groups via View menu.
